doc/source/sparse.rst

```diff
@@ -135,3 +135,83 @@ recommend using ``block`` as it's more memory efficient. The ``integer`` format
 keeps an arrays of all of the locations where the data are not equal to the
 fill value. The ``block`` format tracks only the locations and sizes of blocks
 of data.
+
+Interaction with scipy.sparse
+-----------------------------
+
+Experimental api to transform between sparse pandas and scipy.sparse structures.
+
+:meth:`SparseSeries.to_coo`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A ``SparseSeries.to_coo`` method is implemented for transforming a ``SparseSeries`` indexed by a ``MultiIndex`` to a ``scipy.sparse.coo_matrix``.
+
+The method requires a ``MultiIndex`` with two or more levels.
+
+.. ipython:: python
+   :suppress:
+
+   from numpy import nan
+
+.. ipython:: python
+
+   s = pandas.Series([3.0, nan, 1.0, 3.0, nan, nan])
+   s.index = pandas.MultiIndex.from_tuples([(1, 2, 'a', 0),
+                                            (1, 2, 'a', 1),
+                                            (1, 1, 'b', 0),
+                                            (1, 1, 'b', 1),
+                                            (2, 1, 'b', 0),
+                                            (2, 1, 'b', 1)],
+                                            names=['A', 'B', 'C', 'D'])
+
+   s
+   ss = s.to_sparse() # SparseSeries
+   ss
+
+In the example below, we transform the ``SparseSeries`` to a sparse representation of a 2-d array by specifying that the first and second ``MultiIndex`` levels define labels for the rows and the third and fourth levels define labels for the columns. We also specify that the column and row labels should be sorted in the final sparse representation.
+
+.. ipython:: python
+
+   A, il, jl = ss.to_coo(ilevels=['A', 'B'], jlevels=['C', 'D'],
+                         sort_labels=True)
+
+   A
+   A.todense()
+   il
+   jl
+
+:meth:`SparseSeries.from_coo`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A convenience method for creating a ``SparseSeries`` from a ``scipy.sparse.coo_matrix``.
+
+.. ipython:: python
+   :suppress:
+
+   from numpy import nan
+   import scipy.sparse
```

Comment:
I think the recommended way from scipy it to do `from scipy import sparse`, and then `sparse.coo_matrix` (although the docs seem to use `from scipy.sparse import coo_matrix`